packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -825,6 +831,9 @@ class TimeInput extends React.Component {
             />
           </ButtonGroup>
         )}
+        <div id={formatDescriptionId} className={cx('format-text')} aria-label={`Time Format: ${format}`}>
+          {format}
+        </div>
```

Comment:
Should this be included in the help text or was it meant to be placed under the input? This could cause quite a bit of shifting on a page since this height is increased. Could we end up with alignment issues?

Reply:
I don't think that the misalignment we talked about will happen with this until another wrapper is placed over it with the property `display: inline-block`